projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -1,373 +1,148 @@
-
-		return [
 			{
-				label: __( 'All sources', 'jetpack-forms' ),
-				value: null,
+				name: 'spam',
+				title: getTabTitle( __( 'Spam', 'jetpack-forms' ), totalItemsSpam ),
+				className: 'jp-forms__inbox-tab-item',
+				disabled: isLoading,
+			},
+			{
+				name: 'trash',
+				title: getTabTitle( _x( 'Trash', 'noun', 'jetpack-forms' ), totalItemsTrash ),
+				className: 'jp-forms__inbox-tab-item',
+				disabled: isLoading,
 			},
-			...list,
 		];
-	}, [ sourceFilter ] );
-
-	const listKey = useMemo( () => {
-		return join(
-			map( pick( currentQuery, keys( query ) ), ( value, key ) => `${ key }-${ value }` ),
-			'-'
-		);
-	}, [ currentQuery, query ] );
+	}, [ isLoading, totalItemsInbox, totalItemsSpam, totalItemsTrash ] );
+}
 
-	const showBulkActionsMenu = !! selectedResponses.length && ! loading;
+const Inbox = () => {
+	const [ searchParams, setSearchParams ] = useSearchParams();
+	const urlStatus = searchParams.get( 'status' );
+	const navigate = useNavigate();
+	const tabs = useTabItems();
 
-	const classes = clsx( 'jp-forms__inbox', {
-		'is-response-view': !! currentResponseId,
-		'is-response-animation-reverted': responseAnimationDirection < 0,
-	} );
+	// If a user has no responses yet, redirect them to the landing page.
+	useEffect( () => {
+		if ( config( 'hasFeedback' ) ) {
+			return;
+		}
+		navigate( '/landing' );
+	}, [ navigate ] );
 
-	const title = (
-		<>
-			<span className="title">
-				{ isWpcom() ? __( 'Jetpack Forms', 'jetpack-forms' ) : __( 'Responses', 'jetpack-forms' ) }
-			</span>
-			{ isWpcom() && (
-				<span className="subtitle">
-					{ createInterpolateElement(
-						__(
-							'Collect and manage responses from your audience. <a>Learn more</a>',
-							'jetpack-forms'
-						),
-						{
-							a: (
-								<a
-									href="https://jetpack.com/support/jetpack-blocks/contact-form/"
-									rel="noreferrer noopener"
-									target="_blank"
-								/>
-							),
-						}
-					) }
-				</span>
+	const title = <span className="title">{ __( 'Responses', 'jetpack-forms' ) }</span>;
+
+	const subtitle = (
+		<span className="subtitle">
+			{ createInterpolateElement(
+				__( 'Collect and manage responses from your audience. <a>Learn more</a>', 'jetpack-forms' ),
+				{
+					a: (
+						<a
+							href="https://jetpack.com/support/jetpack-blocks/contact-form/"
+							rel="noreferrer noopener"
+							target="_blank"
+						/>
+					),
+				}
 			) }
-			{ /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */ }
-			<a className="back-button" onClick={ handleGoBack }>
-				<Icon icon={ arrowLeft } />
-				{ __( 'View all responses', 'jetpack-forms' ) }
-			</a>
-		</>
+		</span>
 	);
 
+	const onTabSelect = useCallback(
+		newStatusValue => {
+			setSearchParams( previouSearchParams => {
+				const _serachParams = new URLSearchParams( previouSearchParams );
+				_serachParams.set( 'status', newStatusValue );
+				return _serachParams;
```

Comment:
[serach!](https://en.wikipedia.org/wiki/Sriracha)

```suggestion
				const _searchParams = new URLSearchParams( previouSearchParams );
				_searchParams.set( 'status', newStatusValue );
				return _searchParams;
```

Reply:
eb6f92aef435dd0d89d1332af1a1a894fe96498f :) there was also the "**previou**Search" typo

Reply:
Thanks!